src/components/Iconography/Iconography.stories.mdx

```diff
@@ -0,0 +1,176 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+  ArgsTable,
+  Preview,
+  Description,
+} from "@storybook/addon-docs/blocks";
+import { withDesign } from "storybook-addon-designs";
+
+import { getCategory } from "../../utils/componentCategories";
+import Icon from "../Icons/Icon";
+import { IconColors, IconSizes } from "../Icons/IconTypes";
+
+<Meta
+  title={getCategory("Iconography")}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
+      type: "figma",
+      url:
+        "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Master?node-id=10975%3A16",
+    },
+  }}
+/>
+
+# Iconography
+
+Use the NYPL Design System [Icon](https://nypl.github.io/nypl-design-system/storybook-static/?path=/docs/icon--control-props) component to render and control icons and select NYPL logos as `<svg>` elements within consuming apps. Below are a few example icons.
+
+<Preview>
+  <Story name="Icon Examples">
+    <span style={{ marginRight: "1rem" }}>
+      <Icon
+        decorative
+        size={IconSizes.xxxlarge}
+        name="check"
+        style={{ marginRight: "1rem" }}
+      />
+    </span>
+    <span style={{ marginRight: "1rem" }}>
+      <Icon
+        decorative
+        size={IconSizes.xxxlarge}
+        name="close"
+        style={{ marginRight: "1rem" }}
+      />
+    </span>
+    <span style={{ marginRight: "1rem" }}>
+      <Icon
+        decorative
+        size={IconSizes.xxxlarge}
+        name="error_filled"
+        style={{ marginRight: "1rem" }}
+      />
+    </span>
+    <span style={{ marginRight: "1rem" }}>
+      <Icon
+        decorative
+        size={IconSizes.xxxlarge}
+        name="speaker_notes"
+        style={{ marginRight: "1rem" }}
+      />
+    </span>
+    <span style={{ marginRight: "1rem" }}>
+      <Icon
+        decorative
+        size={IconSizes.xxxlarge}
+        name="logo_nypl"
+        style={{ marginRight: "1rem" }}
+      />
+    </span>
+    <span style={{ marginRight: "1rem" }}>
+      <Icon decorative size={IconSizes.xxxlarge} name="logo_brooklyn" />
+    </span>
+  </Story>
+</Preview>
+
+The DS utilizes the `Filled` icons available in the [Material UI Design System](https://material-ui.com/components/material-icons/).
+
+### Size
+
+Icons in NYPL projects can be rendered at the following sizes.
```

Comment:
Can you write that the sizes are available from the `IconSizes` enum? It's in the preview when you click on "code" but I think being explicit here is good.

Reply:
I added additional verbiage about the enum values.